### ISSUE TITLE
Missing files in web/ folder on OSX

### ISSUE THREAD

**Issue by acouch** [markaspot/mark-a-spot#29]
When I run ``composer install`` on ``dd72b56`` the web folder doesn't have ``index.php`` and other files. Is there an additional step?

```
(mark-a-spot)(master-8.x)$ls -l web
total 0
drwxr-xr-x  33 aaron  staff  1122 Dec 12 11:56 core
drwxr-xr-x  18 aaron  staff   612 Dec 12 11:58 libraries
drwxr-xr-x   3 aaron  staff   102 Dec 12 11:56 modules
drwxr-xr-x   3 aaron  staff   102 Dec 12 11:56 profiles
drwxr-xr-x   3 aaron  staff   102 Dec 12 11:57 themes
```

**Comment by markaspot** [comment]
Have to investigate this. This should be handled by https://github.com/drupal-composer/drupal-scaffold, which should be included in composer.json

**Comment by markaspot** [comment]
I can not really reproduce it. A fresh composer install ends in: 

```
> DrupalProject\composer\ScriptHandler::checkComposerVersion
Loading composer repositories with package information
Updating dependencies (including require-dev)
Writing lock file
Generating autoload files
> DrupalProject\composer\ScriptHandler::createRequiredFiles
Create a sites/default/settings.php file with chmod 0666
Create a sites/default/services.yml file with chmod 0666
Create a sites/default/files directory with chmod 0777
> DrupalProject\composer\ScriptHandler::createRequiredFiles
:~/sites/mark-a-spot$ cd web/
:~/sites/mark-a-spot/web$ l
total 104
 0 drwxr-xr-x  19 sun  staff   646B 12 Dez 18:32 ./
 0 drwxr-xr-x  13 sun  staff   442B 12 Dez 18:32 ../
 8 -rw-r--r--   1 sun  staff   948B 12 Dez 18:32 .csslintrc
 8 -rw-r--r--   1 sun  staff   350B 12 Dez 18:32 .editorconfig
 8 -rw-r--r--   1 sun  staff   185B 12 Dez 18:32 .eslintignore
 8 -rw-r--r--   1 sun  staff    36B 12 Dez 18:32 .eslintrc
 8 -rw-r--r--   1 sun  staff   3,6K 12 Dez 18:32 .gitattributes
16 -rw-r--r--   1 sun  staff   7,7K 12 Dez 18:32 .htaccess
 8 -rw-rw-rw-   1 sun  staff   385B 12 Dez 18:32 autoload.php
 0 drwxr-xr-x  33 sun  staff   1,1K 12 Dez 18:29 core/
 8 -rw-r--r--   1 sun  staff   549B 12 Dez 18:32 index.php
 0 drwxr-xr-x  18 sun  staff   612B 12 Dez 18:32 libraries/
 0 drwxr-xr-x   3 sun  staff   102B 12 Dez 18:29 modules/
 0 drwxr-xr-x   3 sun  staff   102B 12 Dez 18:29 profiles/
 8 -rw-r--r--   1 sun  staff   1,6K 12 Dez 18:32 robots.txt
 0 drwxr-xr-x   6 sun  staff   204B 12 Dez 18:32 sites/
 0 drwxr-xr-x   3 sun  staff   102B 12 Dez 18:30 themes/
 8 -rw-r--r--   1 sun  staff   554B 12 Dez 18:32 update.php
16 -rw-r--r--   1 sun  staff   4,4K 12 Dez 18:32 web.config

```

**Comment by acouch** [comment]
I'll take a further look at Drupal scaffold. I am using OSX 10.10.5. I'll try on a different OS.

**Comment by acouch** [comment]
Works on ubuntu ``16.04``.

**Comment by acouch** [comment]
Updating to a more recent version of drupal-scaffold fixed this for me: #30
